14-draft.json

```diff
@@ -975,6 +975,69 @@
               "value"
             ]
           }
+        },
+        "network_traffic": {
+          "description": "The current network traffic, in bits/second or packets/second (or both)",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "properties": {
+                "type": "object",
+                "properties": {
+                  "bits_per_second": {
+                    "description": "",
+                    "type": "object",
+                    "properties": {
+                      "value": {
+                        "description": "The measurement value, in bits/second",
+                        "type": "number",
+                        "minimum": 0
+                      },
+                      "maximum": {
+                        "description": "The maximum available bandwith in bits/second, e.g. as sold by your ISP",
```

Comment:
Can we switch `bandwidth` with `throughput`? Bandwidth can mean more things and the network guys i've asked use throughput for maximum and current interface usage :)

Reply:
Done